controllers/suite_test.go

```diff
@@ -304,6 +305,37 @@ func newMachine(clusterName, machineName string, metal3machineName string, nodeR
 	return machine
 }
 
+func newMetal3MachineTemplate(clusterName string, m3mTemplateName string, namespace string, isPaused bool) *capm3.Metal3MachineTemplate {
+	template := &capm3.Metal3MachineTemplate{}
+	if isPaused {
```

Comment:
I have even went further now Annotations can be passed to the function so in case no annotation is needed an empty map[string]string can be passed thus there is no need for the if-else control structure.